.github/workflows/license-checker.yml

```
@@ -0,0 +1,22 @@
+name: License checker
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Github has changed the default branch name to `main`.

```suggestion
      - main
  pull_request:
    branches:
      - main
```